mlrun/feature_store/api.py

```diff
@@ -386,6 +394,10 @@ def ingest(
             "featureset.spec.engine must be set to 'spark' to ingest with spark"
         )
     if featureset.spec.engine == "spark":
+        if isinstance(source, pd.DataFrame):
```

Comment:
Why? What about simple ingest with spark?
https://docs.mlrun.org/en/latest/feature-store/transformations.html#using-spark-execution-engine

Reply:
with a df as a source it would fail